ui/src/app/views/profile-registration-step-status/component.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from 'react';
+
+import { Button } from 'app/components/buttons';
+import { ClrIcon } from 'app/components/icons';
+import colors from 'app/styles/colors';
+import { reactStyles } from 'app/utils';
+
+const styles = reactStyles({
+  container: {
```

Comment:
following the existing convention, can you put some of these styles on the same line to reduce the number of lines here?

Reply:
I made the `body` css inline but I kept the rest for the following reasons

container, title - I think its easier to read multiple lines of CSS if they are line by line versus inline.

buttonsContainer, detailsContainer, successButton - i think the class name adds some context on what its being used for. Makes it a quicker read IMO, instead of having to load the UI or read the code to figure out what this div is being used for.

button - its being used twice. Although, it would be nice if I could just apply a style to a tag or children of a tag... Is there a way to do that using `reactStyles`?

Reply:
Discussed and decided separate lines is clear here.